packages/renderer/src/lib/dashboard/NotificationCard.spec.ts

```diff
@@ -0,0 +1,55 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import '@testing-library/jest-dom/vitest';
+import { test, expect, vi, beforeAll } from 'vitest';
+import { fireEvent, render, screen } from '@testing-library/svelte';
+import type { Notification } from '../../../../main/src/plugin/api/notification';
+import NotificationCard from './NotificationCard.svelte';
+
+const removeNotificatioMock = vi.fn();
+
+beforeAll(() => {
+  (window as any).removeNotification = removeNotificatioMock;
```

Comment:
Is there a typo here in removeNotificatioMock => should it be removeNotificatio**n**Mock?